hack/build-cross.sh

```diff
@@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
 kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
 
 KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
+KUBE_STATIC_OVERRIDES=("kubectl")
```

Comment:
It's possible. However, I'd like to find out if there is a requirement for dynamic first. If either is OK I'd say running with static would make sense. If there is a requirement for a dynamic ```kubectl``` along with our new need for a static one then we probably need to update build scripts in such a way that they produce both.